src/pages/testPage/index.tsx

```diff
@@ -37,6 +37,9 @@ export default function TestPage({
     }
     setQuestion(numberOfQuestion + 1);
   }, [numberOfQuestion, questions.length]);
+  const toggleIsRunning = () => {
```

Comment:
Minor thing, but always use blank line between functions. Make the file clearer for the reader

Reply:
done